.travis.yml

```diff
@@ -11,3 +13,10 @@ install:
 
 script:
   - make flake
+  - make test
+
+before_script:
+  # Disable services enabled by default
+  # http://docs.travis-ci.com/user/database-setup/#MySQL
+  - sudo /etc/init.d/postgresql stop
```

Comment:
put here `docker-compose stop` or better `docker-compose down -v` for removing volumes and build

Reply:
It's before script (meaning before testing)